packages/ComponentLibrary/src/components/TextInput/TextInput.constants.ts

```diff
@@ -0,0 +1,13 @@
+export const CONSTANTS = {
```

Comment:
Are these default or mock constants?

If they are default it would be good to rename them better and if they are mock they should be called mock.ts.

Reply:
I agree with him.
Santi had to do the same change